spec/jwt/x5t_key_finder_spec.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+RSpec.describe JWT::X5tKeyFinder do
+  let(:root_key) { test_pkey('rsa-2048-private.pem') }
+  let(:root_dn) { OpenSSL::X509::Name.parse('/DC=org/DC=fake-ca/CN=Fake CA') }
+  let(:root_certificate) { generate_root_cert(root_dn, root_key) }
+  let(:leaf_key) { generate_key }
+  let(:leaf_dn) { OpenSSL::X509::Name.parse('/DC=org/DC=fake/CN=Fake') }
+  let(:leaf_certificate) do
+    cert = generate_cert(leaf_dn, leaf_key.public_key, 2)
+    cert.sign(root_key, 'sha256')
+    cert
+  end
+
+  subject(:keyfinder) { described_class.new([leaf_certificate]).from(header) }
+
+  context 'when certificates argument is nil' do
+    subject(:keyfinder) { described_class.new(nil).from({}) }
```

Comment:
No need to call from in this scenario

```suggestion
    subject(:keyfinder) { described_class.new(nil) }
```